cmd/brew-test-bot.rb

```diff
@@ -721,8 +724,10 @@ def setup_formulae_deps_instances(formula, formula_name)
       build_dependencies = dependencies - runtime_or_test_dependencies
       @unchanged_build_dependencies = build_dependencies - @formulae
 
+      uses_args = []
+      uses_args << "--recursive" unless ARGV.include?("--skip-recursive-dependents")
```

Comment:
Could do something similarly here perhaps? https://github.com/Homebrew/homebrew-test-bot/blob/f7429c951c86edac45ab2e8bc85d410aae0deb1b/cmd/brew-test-bot.rb#L873

Not gonna block this on that suggestion (as it's not necessary but may speed things up).

Reply:
This line runs `brew uses --recursive formula_name` and then ignores the result. Do you know the purpose of this line?

Reply:
Can it be removed altogether?

Reply:
that line was last edited in https://github.com/Homebrew/homebrew-test-bot/pull/21

Since it's at the beginning of `def formula`, I'm guessing it lets you fail fast if there are any problems with formulae?

Reply:
I want back through the commit history for that line, and `brew uses formula_name` goes back to the dawn of time. The `--recursive` option was added, then removed for performance reasons, then added back in. I'm guessing it's to fail fast as well, but since we run `brew readall`, it seems a bit redundant.

Reply:
I suppose we could try removing it in another branch and see if the code coverage changes?

Reply:
@MikeMcQuaid Thoughts?

Reply:
I'm fine with seeing it removed as it's used for testing and being able to query the output rather than `test-bot` doing anything with it. This would be a case where a `brew uses` cache would make sense.

Reply:
I've opened PR https://github.com/Homebrew/homebrew-test-bot/pull/270 to remove this call to `brew uses --recursive`.